CSP: add new information related to security (#5313)
DevExpress/devextreme-documentation: concepts/Common/Localization/05 Localize Dates, Numbers, and Currencies/10 Using Globalize.md

[important] React projects created with the [Create React App](https://facebook.github.io/create-react-app) <a href="https://github.com/facebook/create-react-app/issues/4508" target="_blank">do not support Globalize</a>. [DevExtreme React Template](/concepts/50%20React%20Components/50%20Application%20Template/00%20Application%20Template.md '/Documentation/Guide/React_Components/Application_Template/') is one of such projects. [Use Intl](/concepts/Common/Localization/05%20Localize%20Dates%2C%20Numbers%2C%20and%20Currencies/05%20Using%20Intl.md '/Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies/Using_Intl') in these projects instead.

> [note]If you apply [CSP rules](/Documentation/Guide/Common/Security_Considerations/#Content_Security_Policy), we do not recommend you to use the Globalize approach. Use the [Intl](/Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies/Using_Intl) approach instead.

Reviewer: ```suggestion
[note]If you apply [CSP rules](/Documentation/Guide/Common/Security_Considerations/#Content_Security_Policy), we do not recommend that you use the Globalize technique. Use the [Intl](/Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies/Using_Intl) technique instead.
```

> Activating Globalize in your project requires the following files:

Reviewer: ```suggestion
The following files are required to activate Globalize in your project:
```

> 

Reviewer: Please update the following lines below:

Line 9:
`All the components are available via CDN and npm.`

1. Please change "All the components" to "These components".

2. We should update this sentence without using via, but I think we should clarify it as well. For example, when you say "CDN and npm", do you mean that CDN provider libraries and npm libraries should be used to access these components? Please clarify and update the sentence.

Line 13:
`In this example, German [dictionary](/concepts/Common/Localization/01%20Dictionaries '/Documentation/Guide/Common/Localization/#Dictionaries') is included.`

Please add a definite article ("the") before the word "German".

`Note that the order you include the libraries is important.`
Please change to:

`Note that the order in which you include the libraries is important.`

Line 58:
`These examples include German dictionary.`

Please add a definite article ("the") before the word "German".
